client/src/pages/CreateContest/CreateContestForm/CreateContestForm.tsx

```diff
@@ -0,0 +1,145 @@
+import { useState, useEffect } from 'react';
+import { Button, TextField, Box, Typography, MenuItem, Grid, InputAdornment } from '@material-ui/core';
+import { ImageList, ImageListItem } from '@material-ui/core';
+import { KeyboardDatePicker, KeyboardTimePicker, MuiPickersUtilsProvider } from '@material-ui/pickers';
+import DateRangeIcon from '@material-ui/icons/DateRange';
+import AlarmIcon from '@material-ui/icons/Alarm';
+import AttachMoneyIcon from '@material-ui/icons/AttachMoney';
+import DateFnsUtils from '@date-io/date-fns';
+import useStyles from './useStyles';
+
+export default function CreateContestForm():JSX.Element {
+  const [design, setDesign] = useState<string>('');
+  const [description, setDescription] = useState<string>('');
+  const [prize, setPrize] = useState<string>('');
+  const [date, setDate] = useState<Date | null>(new Date());
+  const [time, setTime] = useState<Date | null>(new Date());
```

Comment:
We can mange the date and time using the `date` state, we don't to have a separate state for time :)